### PR TITLE
Update default config for sh-imessage

### PR DESCRIPTION
**Pull request by ngencokamin**
Updates default number of days to backfill from 0.5 days to 2 weeks, changes param to fix imessage contact names and photos across all platforms. Tested with bluebubbles and applescript connectors

### CODE REVIEW

File: bridgeconfig/imessage.tpl.yaml [hunk 1]
```diff
@@ -218,7 +218,7 @@ bridge:
     caption_in_message: true
     # Should the bridge explicitly set the avatar and room name for private chat portal rooms?
     # This is implicitly enabled in encrypted rooms.
-    private_chat_portal_meta: never
+    private_chat_portal_meta: auto
```
Comment (tulir) [inline bridgeconfig/imessage.tpl.yaml:221]: `auto` is not a valid value, the options are `always`, `default` and `never`

should probably also have all the bridges use the same value 🤔

File: bridgeconfig/imessage.tpl.yaml [hunk 2]
```diff
@@ -153,7 +153,7 @@ bridge:
         # Maximum number of messages to backfill for new portal rooms.
         initial_limit: 100
         # Maximum age of chats to sync in days.
-        initial_sync_max_age: 0.5
+        initial_sync_max_age: 14
```
Comment (tulir) [inline bridgeconfig/imessage.tpl.yaml:156]: Not sure if it should be that high by default, might be better to just add a way to override config fields with `bbctl run` arguments

File: bridgeconfig/imessage.tpl.yaml (hunk 1 above)
Comment (ngencokamin) [reply bridgeconfig/imessage.tpl.yaml:221]: Gotcha, and default will enable this, correct? Also, the ask is there I update all the configs?

File: bridgeconfig/imessage.tpl.yaml (hunk 2 above)
Comment (ngencokamin) [reply bridgeconfig/imessage.tpl.yaml:156]: Gotcha. It's been super Speedy in my experience with Don's changes, but I can for sure set it to like 7 days with a way to override in params?

**Review by tulir — APPROVED** [review]

**Pull request merged**